### ISSUE TITLE
Wiederkehrende Schichten

### ISSUE THREAD

**Issue by s3bb3** [muccc/engelsystem#77]
Zum Anlegen bräuchten wir neben Schichten, die nach einander immer gleich sind, wiederkehrende Schichten, die gleich sind aber nicht nahtlos aneinander.
(Tagsüber immer 20 Übersetzer benötigt, nachts nur 6/Schlafplatzvermittlung nur abends)

**Comment by ElectricMaxxx** [comment]
Hier würde i h gerne dann auch dass MassenEdit unterbringen wollen.  Also so wie Google es macht "nur dieses Event/alle/alle kommenden". Damit haben wir eigentlich schon genug Referenzierung zwischen Schichten  und ich würde ungern noch eine aufmachen.

**Comment by ghost** [comment]
ja, ist glaube ich am schlüssigsten. idealerweise noch ein "XX Schichten betroffen"
